### DISCUSSION TITLE
Link to imported Config

### DISCUSSION THREAD

**Discussion by Arguably-Nutcase** [bartificer/xkpasswd-js#30] [Ideas]
A Link to an already imported Config would be appreciated. Makes it possible for users to easily bookmark their preferred config.

**Comment by podfeet** [comment]
@Arguably-Nutcase sorry for the slow response - I never realized anyone had started a discussion!  To the right it shows you can create an issue from this discussion and then the team can mark this as an enhancement request. I'll do it for you on this one.

We're working on the 1.0 release that won't yet include importing our own configs but it's on the project roadmap.

**Comment by hepabolu** [comment]
@Arguably-Nutcase and @podfeet I am currently working on this feature in issue #14 . This should reintroduce the old way of import/export of config file and the generation of a bookmarkable url as generously donated by Luis Tavares.
I cannot promise a deadline, but know that your suggestion is picked up and will be implemented.

**Comment by hepabolu** [comment]
This feature request is now implemented as the config URL, visible under the Settings box. An alternative option is to save/load a JSON file with your own configuration. 
Thanks for the request. I'll close this discussion now.
